src/Nethermind/Nethermind.AuRa.Test/AuRaSealValidatorTests.cs

```diff
@@ -48,6 +48,8 @@ public class AuRaSealValidatorTests
         private IEthereumEcdsa _ethereumEcdsa;
         private static int _currentStep;
         private IReportingValidator _reportingValidator;
+        private IBlockTree _blockTree;
+        private IValidSealerStrategy _validSealerStrategy;
```

Comment:
strange name, first thing that I see and have no idea what it means ValidSealer (treat as minor / feel free ti ignore)

Reply:
Not sure how to name it better, maybe you have an idea. Adding comments.